doc/migrating-configs.md

```diff
@@ -2,6 +2,162 @@
+
+The 2.2.0 version of the configuration is greatly improved over version 2.1.0, with many new fields and behaviors added to the specification.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### File appending
+
+The `files` section of the config has gained a new field called `append`. When this field is set to `true`, if there's a file at the path then the contents will be appended to the existing file.
+
+```json ignition
+{
+  "ignition": { "version": "2.2.0" },
+  "storage": {
+    "files": [{
+      "filesystem": "root",
+      "path": "/etc/hosts",
+      "append": true,
+      "mode": 420,
+      "contents": {
+        "source": "data:,10.0.0.2%20myname"
+      }
+    }]
+  }
+}
+```
+
+### Node overwriting
+
+The `files`, `directories`, and `links` sections of the config have each gained a new field called `overwrite. When this field is set to `true`, any preexisting nodes at the path of the thing to be created will be overwritten. This field defaults to `true` for files, and `false` for directories and links.
+
+```json ignition
+{
+ "ignition": { "version": "2.2.0" },
+ "storage": {
+   "links": [{
+     "filesystem": "root",
+     "path": "/etc/localtime",
+     "target": "/usr/share/zoneinfo/US/Pacific",
+     "overwrite": true
+   }]
+ }
+}
+```
+
+### Custom raid options
+
+The `raid` section has gained a new field called `options`, that allows arbitrary mdadm arguments to be specified. These arguments are passed directly on to mdadm when raid arrays are being created.
+
+```json ignition
+{
+  "ignition": { "version": "2.2.0" },
+  "storage": {
+    "disks": [
+      {
+        "device": "/dev/sdb",
+        "wipeTable": true,
+        "partitions": [{
+          "label": "raid.1.1",
+          "number": 1,
+          "size": 20480,
+          "start": 0
+        }]
+      },
+      {
+        "device": "/dev/sdc",
+        "wipeTable": true,
+        "partitions": [{
+          "label": "raid.1.2",
+          "number": 1,
+          "size": 20480,
+          "start": 0
+        }]
+      }
+    ],
+    "raid": [{
+      "devices": [
+        "/dev/disk/by-partlabel/raid.1.1",
+        "/dev/disk/by-partlabel/raid.1.2"
+      ],
+      "level": "stripe",
+      "name": "data",
+      "options": [
+        "--verbose"
+      ]
+    }],
+    "filesystems": [{
+      "mount": {
+        "device": "/dev/md/data",
+        "format": "ext4",
+        "label": "DATA"
+      }
+    }]
+  },
+  "systemd": {
+    "units": [{
+      "name": "var-lib-data.mount",
+      "enable": true,
+      "contents": "[Mount]\nWhat=/dev/md/data\nWhere=/var/lib/data\nType=ext4\n\n[Install]\nWantedBy=local-fs.target"
+    }]
+  }
+}
+```
+
+### Custom certificate authorities
+
+The `ignition` section has gained a new section named `security`, which can be used to specify custom certificate authorities to be used when fetching objects over `https`. These are used in addition to the system pool.
```

Comment:
Maybe clarify they aren't added to the system pool outside the initramfs so users don't think they can add CAs to their actual running system with this?